### ISSUE TITLE
Browser spoofing feature

### ISSUE THREAD

**Issue by iliketurtlesbro** [EricssonResearch/bowser#59]
I am attempting to use Bowser with Firefox Hello, but at this time Firefox Hello is seeing my iOS user agent string and giving me an error about iOS not supporting WebRTC. Normally this would be a correct assumption, but thanks to Bowser this is not the case. I plan to send feedback to Mozilla in regards to this, but I also suggest a user agent spoofing feature for Bowser. This will ensure users won't encounter similar issues and be stuck without a solution.

**Comment by stefanalund** [comment]
Yes, this is a recurring problem since various services makes assumptions on capabilities instead of doing feature probing. It is quite easy to change the UserAgent in code (see https://github.com/EricssonResearch/bowser/blob/master/bowser-ios/Bowser/BowserAppDelegate.m#L34) but a better solution would be to have a UI in the app for switching between different options. 

If you are willing and able to help out we would really appreciate it.
